assets/logos/logogebruik_typografie.svg

```diff
@@ -0,0 +1,1018 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Dit en `logogebruik_omgevingsloket.svg` zijn geen assets, het is geen logo, maar het staat wel in de assetes/logo folder